### PR TITLE
fix: add missing files to packages

### PR DESCRIPTION
**Pull request by malangfox**
## Issue
#516 

## Details
This adds a files field to package.json to avoid missing files on publish.

### CODE REVIEW

**Review by daybrush — APPROVED** [review]
LGTM

File: packages/infinitegrid/package.json [hunk 1]
```diff
@@ -29,6 +29,12 @@
 	"namespace": {
 		"eg": "eg"
 	},
+	"files": [
```
Comment (daybrush) [inline packages/infinitegrid/package.json:32]: Add the files field to all frameworks.

**Review by WoodNeck — APPROVED** [review]
LGTM

**Comment by tkmin** [comment]
LGBT

File: packages/infinitegrid/package.json (hunk 1 above)
Comment (malangfox) [reply packages/infinitegrid/package.json:32]: Also I've checked some packages do not have npmignore and newly added.

**Pull request merged**